src/main/java/org/genomicsdb/spark/sources/GenomicsDBBatch.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * The MIT License (MIT)
+ * Copyright (c) 2018 Omics Data Automation
```

Comment:
Should this be `Copyright (c) 2020 Omics Data Automation` or if it has been moved, just update the dates e.g `Copyright © 2018, 2020 Omics Data Automation` or `Copyright © 2018-2020 Omics Data Automation`

Reply:
I went with Copyright (c) 2020.